ui/src/addCriteria.tsx

```diff
@@ -145,7 +146,7 @@ export function AddCriteria() {
       }}
     >
       <Search placeholder="Search criteria or select from the options below" />
-      <ActionBar title={"Add criteria"} />
+      <ActionBar title={"Add criteria"} extraControls={<UndoRedo />} />
```

Comment:
Do you need `UndoRedo` in the `addCriteria` page? Or is this the "edit existing" criteria page and `newCriteria` is the one that adds one?

Reply:
Yes, the `UndoRedo` controls should be present everywhere within cohort editing. `addCriteria` is the page to select a new criteria to add either through searching or selecting from a list. `newCriteria` facilitates editing a temporary new criteria before it has actually been committed, whereas `edit` is for editing a criteria that's already in the data. They all use the `CriteriaHolder` to show the actual criteria editor and are just for sourcing the criteria appropriately.

Reply:
Gotcha, thanks for the explanation.